tests/Auth0.ManagementApi.IntegrationTests/TicketsTests.cs

```diff
@@ -62,7 +62,7 @@ public async Task Test_tickets_sequence()
             {
                 UserId = _user.UserId,
                 ResultUrl = "http://www.nonexistingdomain.aaa/success",
-                NewPassword = Password
+                MarkEmailAsVerified = true
```

Comment:
Would this line fail if the method `MarkEmailAsVerified` does not exists? I think you also added `IncludeEmailInRedirect` and would be nice to include it on the tests.

Reply:
I'll add IncludeEmailInRedirect to this test.  The app would fail to compile if MarkEmailAsVerified was missing.